evap/evaluation/models.py

```diff
@@ -1105,7 +1106,14 @@ class Question(models.Model):
 
     QUESTION_TYPES = (
         (_("Text"), ((QuestionType.TEXT, _("Text question")),)),
-        (_("Unipolar Likert"), ((QuestionType.LIKERT, _("Agreement question")),)),
+        (
+            _("Unipolar Likert"),
+            (
+                (QuestionType.POSITIVE_LIKERT, _("Positive agreement")),
+                (QuestionType.NEGATIVE_LIKERT, _("Negative agreement")),
+            ),
```

Comment:
All other questions have the "question" suffix in their description / user-facing name ("Text question", "Positive yes-no question", ...). This distinguishes them from headings. For consistency, I thus think these should be "Positive agreement question" and "Negative agreement question", or is there any reason not to do that?